src/cf-base.less

```diff
@@ -536,26 +506,20 @@ table {
 
 th,
 td {
-    padding: unit(12px / @base-font-size-px, em) unit(15px / @base-font-size-px, em);
-    background: @td-bg;
+    padding: 10px;
 
     thead & {
         color: @thead-text;
         background: @thead-bg;
-    }
-
-    tbody > tr:nth-child(odd) > & {
-        background: @td-bg-alt;
-    }
-
-    .compact-table & {
-        padding: unit(7px / @base-font-size-px, em) unit(10px / @base-font-size-px, em);
+        font-size: .875em;
+        text-transform: uppercase;
+        .webfont-demi();
```

Comment:
You could use `.h4()` instead of setting these https://github.com/cfpb/cf-core/blob/c3d1227a2bf5235b3a22977d0afd19aa3e2bb965/src/cf-base.less#L226